src/clj/game/core/turns.clj

```diff
@@ -1,24 +1,24 @@
 (ns game.core.turns
   (:require
-    [game.core.agendas :refer [update-all-advancement-requirements]]
```

Comment:
Let's limit the changes to non-cosmetic stuff

Reply:
Oops, overeager committing!